macros/excluded_cloud_binaries.yml

```
@@ -0,0 +1,3 @@
+definition: search binary != "/app/aws-vpc-cni" AND binary != "/usr/bin/amazon-ssm-agent" AND binary != "/usr/bin/ssm-agent-worker"
```

To catch `/usr/bin` and `/bin` 

Also does `/app/aws-vpc-cni` need a wildcard at the start?

```suggestion
definition: search binary != "/app/aws-vpc-cni" AND binary != "*/bin/amazon-ssm-agent" AND binary != "*/bin/ssm-agent-worker"
```